tests/common/jcli_wrapper/mod.rs

```diff
@@ -15,6 +15,12 @@ use super::process_utils::output_extensions::ProcessOutput;
 use std::collections::BTreeMap;
 use std::path::PathBuf;
 
+#[derive(PartialEq)]
+pub enum Discrimination {
+    Production,
+    Test,
+}
+
```

Comment:
Tip: if you'd like to have a clearly designated default value for this, make `impl Default for Discrimination { ... }`, then you can use `Default::default()` as a constructor.

Reply:
Ok. will implement this in next PR